documentation/tutorials/dapr.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: documentation
+menu: tut_dapr
+subtitle: 'Tutorial: Integrating Dapr'
+---
+# Tutorial: Integrating Dapr
+{% include teams.html %}
+
+This brief tutorial will teach you **how to integrate Dapr into your Flyway process**.
+
+## Introduction
+
+Dapr is an application runtime which has a secrets management component, allowing you to securely store and 
+provide access to sensitive information. You can learn more about it 
+[here](https://docs.dapr.io/developing-applications/building-blocks/secrets/secrets-overview/).
+
+Flyway integrates with Dapr's [Secret Store](https://www.vaultproject.io/docs/secrets/kv), letting you 
+securely store and provide access to any confidential Flyway parameters for any specified duration.
```

Comment:
We should probably remove the bit about duration here, as it doesn't seem like Dapr takes an access token?